### ISSUE TITLE
Dark discoloration of nails

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10709]
**Preferred term label:**
Dark discoloration of nails

**Synonyms**

**Definition (free text, please give PubMed ID)**
This is similar to https://hpo.jax.org/browse/term/HP:0100644 but includes also bluish and reddish discoloration.  This could be dealt with by adding a new term or adjusting the existing term.  But this needs to be very distinct from yellow nails.

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0100643

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Subungual nevi
Subungual hematoma
Glomus tumor
Melanoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I think this was what was meant by the term. I have updated the definition as follows:
Dark discoloration of a fingernail or toenail (brown, black, bluish and reddish).
and added this comment
The most serious disease of the nail unit, melanoma, primarily presents with melanonychia. However, melanonychia most often occurs as a result of benign etiologies such as nail matrix melanocytic activation, nail matrix melanocytic hyperplasia, subungual hematoma, glomus tumor, and nail invasion by melanin-producing pathogens.
and added "Dark discoloration of nail" as a synonym
